src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1139,10 +1140,10 @@ public static bool IsPossibleTupleElementNameContext(this SyntaxToken token, int
 
             if (token.IsKind(SyntaxKind.OpenParenToken))
             {
-                return token.Parent.IsKind(SyntaxKind.ParenthesizedExpression) || token.Parent.IsKind(SyntaxKind.TupleExpression);
+                return token.Parent.IsKind(SyntaxKind.ParenthesizedExpression) || token.Parent.IsKind(SyntaxKind.TupleExpression, SyntaxKind.TupleType);
```

Comment:
this doesn't make sense given the method you're in.  If you're after the open paren in a tuple-type then that is  *not* a "PossibleTupleElementNameContext".  Sounds like you might need an IsTupleElementTypeContext or IsPossibleTupleElementTypeContext or something.